src/ServiceBouncer/MainForm.cs

```diff
@@ -15,22 +18,45 @@ public partial class MainForm : Form
         private readonly List<ServiceViewModel> services;
         private bool isActive;
         private string machineHostname;
-        private int backgroundRefreshSeconds = 1;
+        private int backgroundRefreshSeconds;
 
         public MainForm()
         {
             InitializeComponent();
             isActive = true;
+            backgroundRefreshSeconds = 1;
             machineHostname = Environment.MachineName;
             toolStripConnectToTextBox.Text = machineHostname;
             services = new List<ServiceViewModel>();
             Microsoft.Win32.SystemEvents.SessionSwitch += SessionSwitch;
+
 #if NET45
             //In NET45 startup type requires WMI, so it doesn't auto refresh
             dataGridStatupType.HeaderText = $"{dataGridStatupType.HeaderText} (No Auto Refresh)";
 #endif
         }
 
+        public MainForm(Options options)
+        {
```

Comment:
Can you make this a single constructor?
I think some of the logic is missing from original constructor.

Reply:
I can that was the initial way, but when refactoring decided to split. Will do

Reply:
Yeah sometimes it's good to split, but I think in this case, make the main method pass the machine name into ctor. And put the logic into program.cs